support/ci/shared.ps1

```diff
@@ -30,6 +30,10 @@ function Install-Rustup($Toolchain) {
     }
 }
 
+function Get-Rust-Toolchain {
```

Comment:
actually `Get-RustToolchain` (just 1 dash). Might want to make it `Get-Toolchain` to align with the bash function naming which works.

Reply:
Ah, good point. I went with `Rust` in the name to match `Install-RustToolchain` in the same file. Any preference between `Get-RustToolchain` and `Get-Toolchain`?

Reply:
I like `Get-Toolchain`

Reply:
[Done](https://github.com/habitat-sh/habitat/pull/6593/commits/e17eccc6aed51c8486fe901d987d9ba1e4f4e976)!